Gemfile

```diff
@@ -4,7 +4,7 @@ gem 'rails', '~> 4.2.6'
 
 gem 'browserify-rails'
 gem 'coffee-rails', '~> 4.1.0'
-gem 'devise', '~> 3.5'
+gem 'devise', '~> 4.1'
```

Comment:
For gems like Devise that are likely to break the app when a new major version is released, I think pinning makes sense. For example, it makes it easier to use bummr to automate updating gems since we won't have to deal with failure scenarios.